certification/pyxis/pyxis.go

```diff
@@ -47,44 +46,41 @@ func NewPyxisClient(pyxisHost string, apiToken string, projectId string, httpCli
 func (p *pyxisClient) createImage(ctx context.Context, certImage *CertImage) (*CertImage, error) {
 	b, err := json.Marshal(certImage)
 	if err != nil {
-		log.Error(err)
-		return nil, err
+		return nil, fmt.Errorf("could not marshal certImage: %w", err)
```

Comment:
For things that are common across all methods
- `could not marshal`
- `could not read body`
- `could not create request`
- etc

Does it make sense to have `errors` for these? Or at least constants for the strings to reduce duplication?

Reply:
If we're going to bother to add context to the error, we might as well have good info.

Reply:
And I meant "could not marshal certImage". "could not marshal foo". "could not marshal bar". I'm ok with it as is. If you feel strongly we can discuss further.